kerchunk/tests/test_grib.py

```diff
@@ -346,3 +347,45 @@ def test_parse_grib_idx_content(idx_url, storage_options):
         idx_df.iloc[message_no]["length"]
         == output[message_no]["refs"]["longitude/0.0"][2]
     )
+
+
+@pytest.mark.parametrize(
+    "basename, local, storage_options, grib_file",
+    [
+        (
+            "gs://global-forecast-system/gfs.20230928/00/atmos/gfs.t00z.pgrb2.0p25.f000",
+            True,
+            {},
+            "gfs.t00z.pgrb2.0p25.f000.test-limit-10",
+        ),
+        (
+            "s3://noaa-hrrr-bdp-pds/hrrr.20220804/conus/hrrr.t00z.wrfsfcf01.grib2",
+            True,
+            dict(anon=True),
+            "hrrr.t00z.wrfsfcf01.grib2.test-limit-10",
+        ),
+    ],
+)
+def test_make_test_grib_idx_files_local(basename, local, storage_options, grib_file):
```

Comment:
@martindurant I need your help here. I can't figure how this test is failing. I've tried two way but failing on both.